src/Tool-Profilers/SmalltalkImage.extension.st

```diff
@@ -50,36 +50,32 @@ SmalltalkImage >> reportCPUandRAM [
 	SmalltalkImage current reportCPUandRAM
 	"	
 
-	| stream |
 	"VM statistics (Memory use and GC, mainly)"
-	FileStream 
-		forceNewFileNamed: 'Log-MemoryStats.txt'
-		do: [:aStream | aStream nextPutAll: self vm statisticsReport ].
+	'Log-MemoryStats.txt' asFileReference writeStreamDo: [ :aStream | 
+		aStream nextPutAll: self vm statisticsReport ].
 	
 	"Process list"
-	stream := FileStream 
-				forceNewFileNamed: 'Log-ProcessList.txt' 
-				do: [ :aStream |  self tools processBrowser new processNameList 
-								do: [ :each | aStream nextPutAll: each; cr ]].
+	'Log-ProcessList.txt' asFileReference writeStreamDo: [ :aStream |
+		self tools processBrowser new processNameList do: [ :each | 
+			aStream nextPutAll: each; cr ] ].
 
 "Fork all these, so they run in sequence, as the system is back running"
 [
 	"Process taking most CPU"
-	stream := FileStream forceNewFileNamed: 'Log-ThePig.txt'.
-	self tools processBrowser dumpPigStackOn: stream andClose: true.
+	'Log-ThePig.txt' asFileReference writeStreamDo: [ :aStream |
+		self tools processBrowser dumpPigStackOn: aStream andClose: false ].
```

Comment:
Since it's the only user of this method maybe #dumpPigStackOn:andClose: could be transformed into #dumpPigStackOn: only? And it will never close the stream.